ScriptureRenderingPipeline/Models/DownloadLink.cs

```diff
@@ -1,18 +1,12 @@
-﻿using DotLiquid;
+﻿using System.Text.Json.Serialization;

```

Comment:
Nice.  I like using builtin libraries when possible.

Reply:
Yeah I'm becoming a big fan of System.Text.Json.Serialization especially since there are source generators for it so we can take all the reflection overhead out of the picture